server/routes/api.php

```diff
@@ -12,17 +12,33 @@
 | is assigned the "api" middleware group. Enjoy building your API!
 |
 */
+//
+//Route::group(['prefix' => 'post'], function() {
```

Comment:
do not leave commented code when you commit

Reply:
Done, removed